.github/workflows/plugins-test.0.yaml

```diff
@@ -70,6 +70,7 @@ jobs:
           - gson-scenario
           - elasticjob-3.x-scenario
           - springmvc-reactive-scenario
+          - cxf-scenario
```

Comment:
Better move this to `pugins-test.1.yaml` or `plugins-test.2.yaml`.


```shell
$ python3 tools/select-group.py
/Volumes/data/developer/skywalking/.github/workflows/plugins-test.1.yaml 81
/Volumes/data/developer/skywalking/.github/workflows/plugins-test.2.yaml 81
/Volumes/data/developer/skywalking/.github/workflows/plugins-test.3.yaml 95
/Volumes/data/developer/skywalking/.github/workflows/plugins-test.0.yaml 118
```

Reply:
fixed.